src/hypothesis/core.py

```diff
@@ -924,6 +958,14 @@ def wrapped_test(*arguments, **kwargs):
                     '@given(booleans()) @given(integers()), you could write '
                     '@given(booleans(), integers())'
                 ))
+            cls = get_defining_class(test) or object
+            if issubclass(cls, TestCase) and test.__name__ in dir(TestCase):
+                note_deprecation(
+                    'You have applied @given to a %r method, which is used '
+                    'by unittest to run the test rather than being a test '
+                    'in itself.  This is not useful in any way, and will '
+                    'become an error in future.' % test.__name__
```

Comment:
I think it's OK to just make this an error now. It's straightforwardly a usage error and is pretty actively doing nonsense at the moment, and I don't think leaving it enabled helps anyone..

Reply:
According to @doismellburning it's in their codebase, so it's just deprecated until we bump to Hypothesis 4.0 😖   (semantic versioning FTW...)